doc/material-getting-started.md

```diff
@@ -177,7 +177,6 @@ Use this when you want to specify MANUALLY each colors.
                 <Color x:Key="OnSurfaceVariantColor">#A5A0AC</Color>
                 <Color x:Key="SurfaceInverseColor">#313033</Color>
                 <Color x:Key="OnSurfaceInverseColor">#F4EFF4</Color>
-                <Color x:Key="SurfaceTintColor">#5946D2</Color>
```

Comment:
We shouldn't be removing the SurfaceTintColor, just the extensions surrounding TintedBackground and IsTintEnabled.

We should be following what we did [here](https://github.com/unoplatform/uno.toolkit.ui/pull/987/) to replace the tint in the TabBar

Reply:
Oh!!! Got it!